storage_service/storage_service/settings/test.py

```diff
@@ -13,3 +13,23 @@
         "PORT": "",
     },
 }
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'filters': {
+        'require_debug_false': {
+            '()': 'django.utils.log.RequireDebugFalse'
+        }
+    },
+    'handlers': {
+        'null': {
```

Comment:
Console logger might be a good idea, yeah. I was assuming that tests were likely to raise, not silently fail for other reasons, but that might not be a fair assumption.